src/cephadm/cephadmlib/daemons/monitoring.py

```diff
@@ -45,6 +45,7 @@ class Monitoring(ContainerDaemonForm):
             'args': [
                 '--config.file=/etc/prometheus/prometheus.yml',
                 '--storage.tsdb.path=/prometheus',
+                '--query.lookback-delta=1m',
```

Comment:
we should really look at the performance impact of this change. with ceph usually people will have huge number of metrics coming in at regular intervals and if the lookback-delta is being reduced then as i understand it, this can put a load on the targets (by frequently trying to pull newer metrics) and that can cause issues with the performance. Also I seen stuffs like the query latency might also get increased with this. It might not come up in our local testing with docker based environment but it can have potential risks added to it in a real environment. Have you done any assessment on how impactful this would be for us?

Reply:
@aaSharma14 Lets use default value and not change it.